src/NHibernate.Test/NHSpecificTest/NH3755/Fixture.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Linq;
+using NHibernate.Linq;
+using NUnit.Framework;
+
+namespace NHibernate.Test.NHSpecificTest.NH3755
+{
+	[TestFixture]
+	public class Fixture : BugTestCase
+	{
+		protected override void OnSetUp()
+		{
+			using (ISession session = OpenSession())
+			using (ITransaction transaction = session.BeginTransaction())
+			{
+				var c = new Circle { Property1 = "Circle1", Property2 = "Circle2", Property3 = "Circle3"};
+				session.Save(c);
+
+                var s = new Square { Property1 = "Square1", Property2 = "Square2", Property3 = "Square3" };
+                session.Save(c); session.Save(s);
+
+			    var sc1 = new ShapeContainer {Name = "Circle", Shape = c};
+                var sc2 = new ShapeContainer { Name = "Square", Shape = s };
+
+			    session.Save(sc1);
+			    session.Save(sc2);
+
+				session.Flush();
+				transaction.Commit();
+			}
+		}
+
+		protected override void OnTearDown()
+		{
+			using (ISession session = OpenSession())
+			using (ITransaction transaction = session.BeginTransaction())
+			{
+				session.Delete("from System.Object");
+
+				session.Flush();
+				transaction.Commit();
+			}
+		}
+
+        [Ignore]
```

Comment:
is this attribute in addition or replace?  Also, if its a known issue, is there a timeline for the fix? I can assist